app/views/people/_current_roles.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if person.currently_in_a_role? %>
-  <section id="current-roles">
+  <section id="current-roles" dir="<%= page_text_direction %>" lang="<%= person.locale %>">
```

Comment:
This shouldn't be set as right to left, since it could contain 'More about this role' in English.